src/mpi/coll/transports/gentran/gentran_utils.c

```diff
@@ -24,8 +24,9 @@ static inline void vtx_record_issue(MPII_Genutil_vtx_t * vtxp, MPII_Genutil_sche
     LL_APPEND(sched->issued_head, sched->issued_tail, vtxp);
 }
 
-static void vtx_issue(int vtxid, MPII_Genutil_vtx_t * vtxp, MPII_Genutil_sched_t * sched)
+static int vtx_issue(int vtxid, MPII_Genutil_vtx_t * vtxp, MPII_Genutil_sched_t * sched)
```

Comment:
The commit message `coll: fix a warning for gentran ` is not good since it gives no clue what this commit changes. I think if you simply swap the short and long messages it would be much better.

Reply:
fixed commit message